kedro/framework/session/session.py

```diff
@@ -417,6 +417,10 @@ def run(  # pylint: disable=too-many-arguments,too-many-locals
         # Run the runner
         hook_manager = self._hook_manager
         runner = runner or SequentialRunner()
+        if not isinstance(runner, AbstractRunner):
+            raise KedroSessionError(
+                "KedroSession expect an instance of Runner, make sure you are passing an instance instead of a class."
```

Comment:
```suggestion
                "KedroSession expect an instance of Runner, make sure you are passing an instance instead of a class. Have you forgotten the `()` at the end of the statement?"
```

Is this useful or overkill?

Reply:
I'd say let's keep it! Mostly harmless, but can go a long way unpacking the jargon

Reply:
Maybe
```
KedroSession expect an instance of Runner instead of a class. Have you forgotten the `()` at the end of the statement?
```

Reply:
Love it!